stdlib/2/typing.pyi

```diff
@@ -9,13 +9,17 @@ from types import CodeType, FrameType, TracebackType
 overload = object()
 Any = object()
 TypeVar = object()
-Generic = object()
-Tuple = object()
-Callable = object()
-Type = object()
 _promote = object()
 no_type_check = object()
-ClassVar = object()
+
+class _SpecialForm:
+    def __getitem__(self, typeargs: Any) -> Any: ...
+
+Tuple: _SpecialForm = ...
+Generic: _SpecialForm = ...
```

Comment:
> You can't use ``Generic`` in an annotation -- it can _only_ occur in a list of base classes. That's pretty different.

This is exactly what I mean.